src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/CreateStoresActivity/BetterRetailUSA/CreateOrUpdateStore100.json

```diff
@@ -1,26 +1,26 @@
 {
   "number": "BRU-100",
   "isActive": true,
-  "name": "BRU-San Francisco",
```

Comment:
I would keep a store in San Francisco as it could be useful in the future as it is in a different timezone.